dapps/frenemies/src/components/Validators/Balance.tsx

```diff
@@ -1,18 +1,39 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import BigNumber from 'bignumber.js';
+import { SUI_TYPE_ARG } from "@mysten/sui.js";
+import { useWalletKit } from "@mysten/wallet-kit";
+import { useQuery } from "@tanstack/react-query";
+import provider from "../../network/provider";
+
 export function Balance() {
+  const { currentAccount } = useWalletKit();
+  const { data } = useQuery(
+    ["account", "balance"],
+    async () => {
+      const { decimals }= await provider.getCoinMetadata(SUI_TYPE_ARG);
+      const { totalBalance } = await provider.getBalance(currentAccount!, SUI_TYPE_ARG);
+      return new BigNumber(totalBalance).shiftedBy(-1 * decimals).toFormat();
```

Comment:
I think we have a SUI formatting function in Capys, I can pull it here. Not to introduce unnecessary dependency (like we did with `bcs` library).

Reply:
I'd honestly rather keep this here, I think it's fine to have a dep for this.